### DISCUSSION TITLE
Support 1.20.2

### DISCUSSION THREAD

**Discussion by aseCz** [EssentialsX/Essentials#5607] [Q&A]
Hello,
i would like to ask about supporting version 1.20.2. I found, that was released version supports 1.20.2 in 29.9.2023, but after that was released another version supports 1.20.4 and that version is not supports 1.20.2 anymore. Is somewhere version released in september, what i could use it?
Thanks a lot,
Ase

**Comment by pop4959** [comment]
There are no plans to support outdated minor versions of Minecraft. Some of the old development builds support 1.20.2 but you should try to update to 1.20.4 if possible.
